py3status/modules/hddtemp.py

```diff
@@ -133,9 +135,11 @@ def post_config_hook(self):
         self.keys = ["path", "name", "temperature", "unit"]
         self.cache_names = {}
         self.thresholds_init = self.py3.get_color_names_list(self.format_hdd)
+        if not self.py3.check_commands("netcat"):
```

Comment:
For me, nc is symblink to netcat. (manjaro/archlinux).
In debian systems they are linked to alternatives.
`valdur@x /bin> ls -lah ncat nc netcat
lrwxrwxrwx 1 root root   20 juuli 12  2020 nc -> /etc/alternatives/nc*
-rwxr-xr-x 1 root root 221K jaan  30  2023 ncat*
lrwxrwxrwx 1 root root   24 juuli 12  2020 netcat -> /etc/alternatives/netcat*
`
`valdur@x :/etc/alternatives $ ls -lah nc netcat
lrwxrwxrwx 1 root root 20 juuli 12  2020 nc -> /bin/nc.openbsd
lrwxrwxrwx 1 root root 20 juuli 12  2020 netcat -> /bin/nc.openbsd
`
I didn't even know that ncat exists.
For me it looks like netcat/nc is more commonly installed that ncat.

Changed nc to netcat because if you google just nc you wont get far.

Reply:
Interesting. I'm using [Fedora i3 Spin](https://fedoraproject.org/spins/i3/) VM to sometimes deal with py3status on macOS.

Netcat (classic) is not installed, but nc / ncat (nmap) is...

```
lasers@fedora:~$ rpm -qf $(which nc)
nmap-ncat-7.92-2.fc40.aarch64

lasers@fedora:~$ rpm -qf $(which ncat)
nmap-ncat-7.92-2.fc40.aarch64

lasers@fedora:~$ rpm -qf $(which netcat)
/usr/bin/which: no netcat in (/home/lasers/.local/bin:/home/lasers/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin)
rpm: no arguments given for query
```

I think you had it right the first time.... Most linux distributions would set up symlinks pointing to `nc` for most thing... so we could just point it to `nc` too... and just skip `STRING_NOT_FOUND` entirely.

I can't test hddtemp because of VM (vda instead of sda).

Reply:
No, my bad. I would like to check for `nc`. Let's just leave that.